library/Kubernetes/Web/Conditions.php

```diff
@@ -31,53 +28,10 @@ protected function assemble(): void
     {
         $this->addHtml(new HtmlElement('h2', null, new Text($this->translate('Conditions'))));
 
-        $listItems = [];
-
         $conditions = yield_iterable($this->getConditions());
```

Comment:
`$this->getConditions())` can be directly used now in the `new ConditionList` call.